### ISSUE TITLE
Active points color contrast violation

### ISSUE THREAD

**Issue by rmmayo** [NationalSecurityAgency/skills-service#2717]
![image](https://github.com/user-attachments/assets/bb128219-2d25-489c-9f49-fc17f2ec2e31)

**Comment by dwalizer** [comment]
:+1:
